src/translations/en.json

```diff
@@ -479,6 +491,7 @@
   "LandingPage.schemaDescription": "An online marketplace.",
   "LandingPage.schemaTitle": "{marketplaceName}",
   "LayoutWrapperAccountSettingsSideNav.contactDetailsTabTitle": "Contact details",
+  "LayoutWrapperAccountSettingsSideNav.deleteAccountTabTitle": "Account management",
```

Comment:
The key should be different
`LayoutWrapperAccountSettingsSideNav.manageAccountTabTitle`